example/20-componentconfig-gardenlet.yaml

```
@@ -29,6 +29,8 @@ controllers:
     syncPeriod: 30s
   controllerInstallationRequired:
     concurrentSyncs: 1
+  seedApiserverNamespace:
```

Can we rename this controller to `SeedAPIServerNetworkPolicy` controller?
```suggestion
  seedAPIServerNetworkPolicy:
```